api/tls_context.proto

```diff
@@ -41,6 +41,16 @@ message TlsCertificate {
   repeated DataSource signed_certificate_timestamp = 5;
 }
 
+message TlsSessionTicketKeys {
+  // Optional identifier for this set of keys
+  string name = 1;
+
+  // Keys to encrypt/decrypt TLS session tickets for session resumption.  The first
+  // key is used to encrypt new tickets that are created.  All keys are candidates
+  // for decrypting received tickets.
```

Comment:
Should I document the expected format here?  The only reason it seems weird is because if the format ever changes (due to different crypto being used), it seems weird to modify an existing protobuf field (even though the modification is only documentation, and doesn't break the protobuf wire format).

Reply:
I think the format should be documented here, even though it doesn't affect the wire format.